site/app/templates/polls/NewPollPage.twig

```diff
@@ -0,0 +1,246 @@
+<div class="content">
+    {% if not poll is defined %}
+        <h1> Add a New Poll </h1>
+
+        <form method="post" action="{{base_url}}/newPoll">
+            <input type="hidden" name="csrf_token" value="{{ csrf_token }}"/>
+            <input type="hidden" name="response_count" id="response-count" value="0"/>
+
+            <label for="poll-name">
+                <h3> Poll Name: </h3>
+            </label>
+            <input type="text" name="name" id="poll-name" placeholder="Enter name here..."/>
+            <br/> <br/>
+
+            <label for="poll-question">
+                <h3> Question: </h3>
+            </label>
+            <textarea
+                id="poll-question"
+                name="question"
+                style="resize: none; min-height: 100px; max-height: 300px; height: 98px; width: -webkit-fill-available;"
+                rows="10" cols="30"
+                placeholder="Enter question here..."
+                maxlength="5000"></textarea>
+            <br/><br/>
+
+            <label for="poll-time">
+                <h3> Expected release date: </h3>
+            </label>
+            <input type="date" name="release_date" id="poll-date"/>
```

Comment:
On other submitty pages we use the `flatpickr` date pickers. You might want to use that to stay consistent.